apps/crn-server/serverless.ts

```diff
@@ -541,13 +547,13 @@ const serverlessConfig: AWS = {
           eventBridge: {
             eventBus: 'asap-events-${self:provider.stage}',
             pattern: {
-              source: [eventBusSource],
+              source: [eventBusSourceSquidex],
               'detail-type': [
                 'TeamsPublished',
                 'TeamsUpdated',
                 'TeamsUnpublished',
                 'TeamsDeleted',
-              ],
+              ] satisfies WebhookDetailType[],
```

Comment:
Is there a reason this line has been added only here and not to the equivalent place on each handler?